tmt/steps/provision/testcloud.py

```diff
@@ -611,9 +619,11 @@ def prepare_config(self) -> None:
         self.config.DOWNLOAD_PROGRESS = self.debug_level > 2
         self.config.DOWNLOAD_PROGRESS_VERBOSE = False
 
+        testcloud_data = get_workdir_root(self.opt('workdir-root'))
```

Comment:
`testcloud` plugin does not have `workdir-root` option, and it should not really have one, or even care about one. At this point of time, the workdir root has been already established by tmt, and used to spawn `self.workdir`, for example. Instead of guessing what the right value is - with all the involved inputs - we should try to access this single source of truth. We can save the workdir root besides the `self.workdir` attribute, or we can take `self.workdir` and use `.parent` as needed.

And as a concept shared by all plugins, and actually steps and plans as well, there should be one implementation, plugin-agnostic. Putting it into a single plugin does not seem correct to me. Seems more like a property or attribute of `tmt.utils.Common`, for all classes to use when needed.